documentation/man/features/kw-signature.rst

```diff
@@ -0,0 +1,94 @@
+===================================================
+kw-signature - Write commit and patch trailer lines
```

Comment:
I was thinking about the name, and at the end of the day, you are just adding kernel tags to the commit. We can try to find a name around it. Some brainstorm ideas:
* add-commit-tags
* add-tag
* add-signature-tag
* Link -> This is a special case requiring a link, not an email.

Reply:
Ok, it is time for us to end this PR. In this sense, let's rename this feature to `kernel-tags`.